ParseBool documentation incorrect

**Description**

The documentation for [parseBool](https://github.com/hashicorp/levant/blob/master/docs/templates.md#parsebool) seem to be incorrect, or I'm doing something wrong.

**Relevant Nomad job specification file**

not working:
```
[[ if key "prod" | parseBool ]]                                                                                                                                                                                      
    update {                                                                                                                                                                                                         
      max_parallel = 1                                                                                                                                                                                               
      canary = 1                                                                                                                                                                                                     
    }                                                                                                                                                                                                                
[[ end ]]
```
working:
```
[[ if .prod | parseBool ]]                                                                                                                                                                                           
    update {                                                                                                                                                                                                         
      max_parallel = 1                                                                                                                                                                                               
      canary = 1                                                                                                                                                                                                     
    }                                                                                                                                                                                                                
[[ end ]]
```

**Output of `levant version`:**

```
Levant v0.2.8
Date: 2019-09-14T14:09:17Z
Commit: dee2715dbc6bd4c8c3d73c882e017e380782e1d2
Branch: 0.2.8
State: 0.2.8
Summary: dee2715dbc6bd4c8c3d73c882e017e380782e1d2
```

**Output of `consul version`:**

```
Consul v1.8.3
Revision a9322b9c7
Protocol 2 spoken by default, understands 2 to 3 (agent will automatically use protocol >2 when speaking to compatible agents)
```

**Output of `nomad version`:**

```
Nomad v0.12.3 (2db8abd9620dd41cb7bfe399551ba0f7824b3f61)
```

**Additional environment details:**

**Debug log outputs from Levant:**

```
$ levant render -var prod=true -var job_name=some-job-name -var service_name=some-service-name -var image=some-image-name some-nomad-file.nomad                                                                                                                                                                                      
{"level":"debug","time":"2020-09-04T08:47:31+02:00","message":"template/render: no variable file passed, trying defaults"}                                                                                           
{"level":"debug","time":"2020-09-04T08:47:31+02:00","message":"helper/files: no default var-file found"}                                                                                                             
{"level":"info","time":"2020-09-04T08:47:31+02:00","message":"helper/variable: using command line variable with key prod and value true"}                                                                            
{"level":"info","time":"2020-09-04T08:47:31+02:00","message":"helper/variable: using command line variable with key job_name and value some-job-name"}                                                                      
{"level":"info","time":"2020-09-04T08:47:31+02:00","message":"helper/variable: using command line variable with key service_name and value some-service-name"}                                                                  
{"level":"info","time":"2020-09-04T08:47:31+02:00","message":"helper/variable: using command line variable with key image and value some-image-name"}         
[ERROR] levant/command: template: jobTemplate:8:15: executing "jobTemplate" at <parseBool>: error calling parseBool: strconv.ParseBool: parsing "prod": invalid syntax
```

--- a/docs/templates.md
+++ b/docs/templates.md
@@ -224,7 +224,7 @@ Takes the given string and parses it as a boolean value which can be helpful in 
 
 Example:
 ```
-[[ if key "service/config/beta" | parseBool ]][[ "beta-release" ]][[ end ]]
+[[ if "true" | parseBool ]][[ "beta-release" ]][[ end ]]
 ```
 
 Render:
@@ -369,7 +369,7 @@ Takes the argument as a string and converts it to lowercase.
 
 Example:
 ```
-[[ key "service/config/queue-name" | toLower ]]
+[[ "QUEUE-NAME" | toLower ]]
 ```
 
 Render:
@@ -383,7 +383,7 @@ Takes the argument as a string and converts it to uppercase.
 
 Example:
 ```
-[[ key "service/config/queue-name" | toUpper ]]
+[[ "queue-name" | toUpper ]]
 ```
 
 Render:

